app/controllers/katello/api/v2/subscriptions_controller.rb

```diff
@@ -49,9 +49,7 @@ def show
     end
 
     def available
-      subscriptions = if @system
-                        available_for_system
-                      elsif @activation_key
+      subscriptions = if @activation_key
```

Comment:
yes, this was a curious discovery  I'm not 100% sure the history around that as you say it is not documented.  It is listed in the routes:  https://github.com/Katello/katello/blob/master/config/routes/api/v2.rb#L50  which is why i was hesitant to completely remove it.  

looks like the lack of documentation is related to https://github.com/Katello/katello/commit/cbca266af2e5391a6da7023f52131da99f47795e#diff-937e4184d63313d2975db7d87ebd1c90L91

@thomasmckay you have any thoughts?